api/v1alpha/src/index.js

```diff
@@ -113,6 +114,14 @@ var routes = [];
 // CORS will be controlled by the API GW layer
 router.all('*', cors());
 
+if (
+    (process.env.NODE_ENV === 'production' ||
+        process.env.VUE_APP_APICLIENT == 'server') &&
+    process.env.SIMPLE_AUTHENTICATION === 'true'
```

Comment:
Added check for SIMPLE_AUTHENTICATION, only enabling checkIfAuthenticated if the env var is specified.